e2e/nomostest/testpredicates/predicates.go

```diff
@@ -441,7 +440,7 @@ func IsManagedBy(scheme *runtime.Scheme, scope declared.Scope, syncName string)
 		gvk := obj.GetObjectKind().GroupVersionKind()
 		if gvk.Empty() {
 			var err error
-			gvk, err = apiutil.GVKForObject(obj, scheme)
+			gvk, err = kinds.Lookup(obj, scheme)
```

Comment:
These Lookups can be pulled out to a new PR. They're nice to have, but not required for this PR any more.